nixos/modules/services/x11/window-managers/awesome.nix

```diff
@@ -11,6 +11,15 @@ let
     " --search " + (getLuaPath path "share") +
     " --search " + (getLuaPath path "lib")
   );
+  awesomeWrapped = pkgs.buildEnv { 
+    name = "awesome-wrapper"; 
+    paths = [ (writeScriptBin "awesome" ''
+      exec ${awesome}/bin/awesome ${lib.optionalString cfg.noArgb "--no-argb"} ${makeSearchPath cfg.luaModules} "$@"
+    '')
```

Comment:
You are missing a semicolon here therefore eval fails.

Reply:
Thank you for the hint! @lucc